routing/route.hpp

```diff
@@ -134,6 +163,22 @@ class Route
     Update();
   }
 
+  // Subroute interface.
+  /// \returns Number of subroutes.
+  /// \note Intermediate points separate a route into several subroutes.
```

Comment:
However let't leave intermediate. As I found out we'd had everywhere in our code intermediate instead of via.